tests/test_kiwi.py

```diff
@@ -55,7 +55,10 @@ def test_kiwi_installation(auto_container):
     reason="test is atm x86_64 specific",
 )
 @pytest.mark.skipif(
-    PODMAN_SELECTED and os.geteuid() != 0,
+    DOCKER_SELECTED, reason="Kiwi containers are only supported with podman"
```

Comment:
what's the reason for excluding docker? any rationale? there is nothing in the commit message, no reference here in the reason either. I was searching my emails and slack, and couldn't find anything. why are we doing this?

Reply:
the previous condition was trying to avoid rootless podman. both docker and rootful podman are supposed to work from what I can say..

Reply:
maybe you want https://github.com/SUSE/BCI-tests/pull/610 instead ?